recipes/mpi4py/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version="2.0.0" %}
+
+package:
+  name: mpi4py
+  version: {{ version }}
+
+source:
+  fn: mpi4py-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/m/mpi4py/mpi4py-{{ version }}.tar.gz
+  md5: 4f7d8126d7367c239fd67615680990e3
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - python
+    - msmpi          # [win]
+    - openmpi        # [unix]
+  run:
+    - python
+    - msmpi          # [win]
+    - openmpi        # [unix]
+
+test:
+  imports:
+    - mpi4py
+    - mpi4py.MPI
+
+about:
+  home: http://mpi4py.scipy.org/
```

Comment:
Got a forbidden message when trying to access this URL. Not sure if http://pythonhosted.org/mpi4py/ is more up-to-date.